## Python Data/Hora 

### Datas Python 

Uma data em Python não é um tipo de dados prório, mas podemos importar um módulo nomeado para trabalhar com datas como data Objetos. 
`datetime`

Importe o módulo datetime e exiba a data atual:

In [10]:
import datetime 

data_hora  = datetime.datetime.now()
print(data_hora)

2024-07-11 16:37:11.144441


## Data de saída 

Quando executamos o código do exemplo acima o resultado será: 

`2024-07-11 16:37:11.144441` 

A data contém ano, mês, dia, hora, minuto e microssegundo. 

O móduilo tem muitos métodos para retornaer informações sobre a data objeto `datetime:`

Devolva o ano e nome do dia da semana: 


In [11]:
data_hora = datetime.datetime.now()

print(data_hora.year)
print(data_hora.strftime('%A'))


2024
Thursday


## Criando objetos de data

Para criar uma data, podemos usar a classe `datetime()` (construtor) do módulo `datetime`.

A classe `datetime()` requer três parâmetros para criar uma data: ano, mês, dia.

      
Crie um objeto de data: 



In [17]:
dh = datetime.datetime(2024, 7, 18)
print(dh)

2024-07-18 00:00:00


A classe `datetime() 0 None` também usa parâmetros para hora e fuso horário(hora, minuto, segundo, microssegundo, itzone), mas eles são opcionais e tem um padradrão valor de, (para fuso horário). 